pkg/apis/garden/v1beta1/conversions.go

```diff
@@ -2244,6 +2256,20 @@ func Convert_v1beta1_ShootStatus_To_garden_ShootStatus(in *ShootStatus, out *gar
 		return err
 	}
 
+	if in.LastError != nil {
+		outLastErrors := []garden.LastError{
+			garden.LastError{
```

Comment:
you don't need to write garden.LastError here. 
```suggestion
			{
```